Makefile

```diff
@@ -2,8 +2,6 @@ MAKEFLAGS = -j1
 FLOW_COMMIT = 622bbc4f07acb77eb1109830c70815f827401d90
 TEST262_COMMIT = 52f70e2f637731aae92a9c9a2d831310c3ab2e1e
 
-export BABEL_ENV = test
```

Comment:
I think we'll want to set this down on line 72 and 82?

Reply:
Yep, caught that too.. just updated!